containers/python/mxnet_container.py

```diff
@@ -0,0 +1,122 @@
+from __future__ import print_function
+import rpc
+import os
+import sys
+import json
+
+import numpy as np
+import cloudpickle
+import mxnet as mx
+import importlib
+
+IMPORT_ERROR_RETURN_CODE = 3
+
+MXNET_MODEL_RELATIVE_PATH = "mxnet_model"
+
+
+def load_predict_func(file_path):
+    with open(file_path, 'r') as serialized_func_file:
+        return cloudpickle.load(serialized_func_file)
+
+
+# load mxnet model from serialized dir
+def load_mxnet_model(model_path):
+    data_iter = mx.io.CSVIter(
```

Comment:
The container should not be loading any training data or doing any training. See next comment.